app/views/about/privacy.en.html.erb

```diff
@@ -0,0 +1,150 @@
+<% title t('.page_title') %>
```

Comment:
If you use the hardcoded 'Privacy policy' string (as the whole page is hardcoded), you don't even need the locales file. But is fine either way 👍

Reply:
I've left this as is, but primarily because I couldn't get it to work -very happy to change it if appropriate

Reply:
I think this would work:

```erb
<% title 'My page title' %>
```

But it is fine either way.

Reply:
I'll switch it. Better to have less code!